Traceback (most recent call last):
  File "fuzz.py", line 23, in <module>
    fuzz()
  File "fuzz.py", line 13, in fuzz
    re.compile(data_bytes)
  File "re.py", line 234, in compile
    return _compile(pattern, flags)
  File "re.py", line 286, in _compile
    p = sre_compile.compile(pattern, flags)
  File "sre_compile.py", line 783, in compile
    groupindex, tuple(indexgroup)
RuntimeError
